doc/content/examples/example3.md

```diff
@@ -87,4 +87,15 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+This example can also be run using the Automatic Differentiation (AD). Automatic Differentiation
```

Comment:
Please expand this. Provide guidance for how the AD input file needs to be built, what classes to use, what solver type, etc. Add links to these classes as well. Please make this a different subsection.